src/ripple/core/JobQueue.h

```diff
@@ -108,7 +108,7 @@ class JobQueue
     using JobFunction = std::function <void(Job&)>;
 
     JobQueue (beast::insight::Collector::ptr const& collector,
-        Stoppable& parent, beast::Journal journal, Logs& logs);
+        Stoppable& parent, beast::Journal journal, Logs& logs, bool validator);
```

Comment:
Nit: It would be nice to pass this to the `setThreadCount` call rather than adding a member that won't be used much. Or set the number of threads in the constructor as in #2136.